### PR TITLE
Styled the sideBar that shows the bookings

### PR DESCRIPTION
**Pull request by javiermijangosksquare**

### CODE REVIEW

File: src/components/BookingsSideBar/BookingList/BookingItem/BookingDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  }
+});
+
+const BookingDetailsComponent = props => {
+  const { classes } = props;
+  return (
+    <Grid container direction={"row"}>
+      <div className={classes.userInfo}>{props.userName}</div>
+      <div className={classes.hourInfo}>
+        {"From " + props.startTime + " to " + props.endTime}
```
Comment (mishelashala) [inline src/components/BookingsSideBar/BookingList/BookingItem/BookingDetails.jsx]: favor `string interpolation` over `string concatenation`

File: src/components/BookingsSideBar/BookingList/BookingItem/BookingItem.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+const normaliceTZ = date => date.substring(0, date.length - 1);
+
+function BookingItemComponent(props) {
+  const { classes } = props;
+  const { roomColor, roomName, userName, start, end } = props.booking;
+  const startDate = moment(normaliceTZ(start));
+  const startTime = startDate.format("LT");
+  const endTime = moment(normaliceTZ(end)).format("LT");
+
+  const getDateText = date => {
```
Comment (mishelashala) [inline src/components/BookingsSideBar/BookingList/BookingItem/BookingItem.jsx]: extract the function outside the component.

File: src/components/BookingsSideBar/BookingList/BookingItem/BookingItem.jsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    if (date) {
+      const today = moment();
+      const tomorrow = moment().add(1, "days");
+      const isCurrentMonth =
+        date.year() === today.year() && date.month() === today.month();
+
+      return isCurrentMonth
```
Comment (mishelashala) [inline src/components/BookingsSideBar/BookingList/BookingItem/BookingItem.jsx]: extract logic into a function

```js
// isCurrentMonth :: Date -> Boolean
```

File: src/components/BookingsSideBar/BookingList/BookingList.jsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  state = {
+    bookingItems: []
+  };
+
+  getBookings = async () => {
+    let bookingItems = this.state.bookingItems.map(async book => {
+      const user = this.props.userService.getUser(book.user_id);
+      const room = this.props.roomService.getRoom(book.room_id);
+      const data = await Promise.all([user, room]);
+
```
Comment (basultobdKs) [inline src/components/BookingsSideBar/BookingList/BookingList.jsx]: I suggest consider the array destructuring.
```js
const [ user, room ] = data;
const { name: userName } = user;
const { name: roomName, color: roomColor } = room;
```

File: src/components/BookingsSideBar/BookingList/BookingList.jsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    this.setState({ bookingItems: items });
+  };
+
+  async componentDidMount() {
+    const data = await this.props.booking.getDetailedListOfBooking();
+    this.setState({ bookingItems: data }, () => this.getBookings());
+  }
+
+  render() {
+    const { classes } = this.props;
```
Comment (basultobdKs) [inline src/components/BookingsSideBar/BookingList/BookingList.jsx]: Also the destructuring issue

File: src/components/BookingsSideBar/BookingsSideBar.jsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+  title: {
+    marginTop: 25,
+    marginLeft: 50,
+    fontSize: "1.8em",
+    fontWeight: "bold",
+    color: "gray"
+  }
+});
+
+const BookingsSideBarComponent = props => {
```
Comment (basultobdKs) [inline src/components/BookingsSideBar/BookingsSideBar.jsx]: Also the destructuring issue

File: src/components/BookingsSideBar/SearchBar.jsx [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+  },
+  input: {
+    fontSize: 18
+  },
+  iconButton: {
+    borderRadius: 5
+  }
+});
+
+const SearchBarComponent = props => {
```
Comment (basultobdKs) [inline src/components/BookingsSideBar/SearchBar.jsx]: Also the destructuring issue

**Pull request merged**